<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StereoCameraView</class>
 <widget class="QWidget" name="StereoCameraView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>双目相机</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>43</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Save To:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_SaveTo"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_FindSaveTo">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>计数器:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="font">
         <font>
          <family>Adobe Devanagari</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="digitCount">
         <number>6</number>
        </property>
        <property name="mode">
         <enum>QLCDNumber::Dec</enum>
        </property>
        <property name="value" stdset="0">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QCameraViewfinder" name="viewfinderL" native="true">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>240</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QCameraViewfinder" name="viewfinder_R" native="true">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>240</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>130</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frame_5">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>130</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>L摄像头:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>L分辨率:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_Lresolution"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>图片格式:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="comboBox_LCodec"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>质量:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="horizontalSlider_LQuality">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>曝光补偿:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QSlider" name="horizontalSlider_LexposureCompensation">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QComboBox" name="comboBox_Lcam"/>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>130</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>R摄像头:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>R分辨率:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_Rresolution"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>图片格式:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="comboBox_RCodec"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>质量:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="horizontalSlider_RQuality">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>曝光补偿:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QSlider" name="horizontalSlider_RexposureCompensation">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QComboBox" name="comboBox_Rcam"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_OpenCamera">
        <property name="text">
         <string>开启摄像头</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_CloseCamera">
        <property name="text">
         <string>关闭摄像头</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_RefreshCam">
        <property name="text">
         <string>刷新摄像头</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Shot">
        <property name="text">
         <string>拍照</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header>qcameraviewfinder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_FindSaveTo</sender>
   <signal>clicked()</signal>
   <receiver>StereoCameraView</receiver>
   <slot>OnFindSavePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>899</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_Lcam</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>StereoCameraView</receiver>
   <slot>OnSelectedChanged_LCameraDevice(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_Lresolution</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>StereoCameraView</receiver>
   <slot>OnSelectedChanged_LResolution(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>595</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_LCodec</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>StereoCameraView</receiver>
   <slot>OnSelectedChanged_LCodec(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_LQuality</sender>
   <signal>valueChanged(int)</signal>
   <receiver>StereoCameraView</receiver>
   <slot>OnChanged_LQuality(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>622</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_LexposureCompensation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>StereoCameraView</receiver>
   <slot>OnChanged_LExposureCompensation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>632</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_Rcam</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>StereoCameraView</receiver>
   <slot>OnSelectedChanged_RCameraDevice(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>942</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>969</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_Rresolution</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>StereoCameraView</receiver>
   <slot>OnSelectedChanged_RResolution(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>730</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>969</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_RCodec</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>StereoCameraView</receiver>
   <slot>OnSelectedChanged_RCodec(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>856</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>970</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_RQuality</sender>
   <signal>valueChanged(int)</signal>
   <receiver>StereoCameraView</receiver>
   <slot>OnChanged_RQuality(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>623</y>
    </hint>
    <hint type="destinationlabel">
     <x>972</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_RexposureCompensation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>StereoCameraView</receiver>
   <slot>OnChanged_RExposureCompensation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>910</x>
     <y>637</y>
    </hint>
    <hint type="destinationlabel">
     <x>971</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Shot</sender>
   <signal>clicked()</signal>
   <receiver>StereoCameraView</receiver>
   <slot>OnShot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>994</x>
     <y>680</y>
    </hint>
    <hint type="destinationlabel">
     <x>973</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_CloseCamera</sender>
   <signal>clicked()</signal>
   <receiver>StereoCameraView</receiver>
   <slot>OnCloseCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>852</x>
     <y>678</y>
    </hint>
    <hint type="destinationlabel">
     <x>971</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_OpenCamera</sender>
   <signal>clicked()</signal>
   <receiver>StereoCameraView</receiver>
   <slot>OnOpenCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>786</x>
     <y>682</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>695</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnFindSavePath()</slot>
  <slot>OnShot()</slot>
  <slot>OnChanged_LExposureCompensation(int)</slot>
  <slot>OnChanged_LQuality(int)</slot>
  <slot>OnSelectedChanged_LCameraDevice(QString)</slot>
  <slot>OnSelectedChanged_LResolution(QString)</slot>
  <slot>OnSelectedChanged_LCodec(QString)</slot>
  <slot>OnChanged_RExposureCompensation(int)</slot>
  <slot>OnChanged_RQuality(int)</slot>
  <slot>OnSelectedChanged_RCameraDevice(QString)</slot>
  <slot>OnSelectedChanged_RResolution(QString)</slot>
  <slot>OnSelectedChanged_RCodec(QString)</slot>
  <slot>OnCloseCamera()</slot>
  <slot>OnOpenCamera()</slot>
 </slots>
</ui>
